Content.Server/Chat/Managers/ChatManager.cs

```diff
@@ -131,19 +132,22 @@ public void EntitySay(EntityUid source, string message, bool hideChat=false)
 
             message = message.Trim();
 
-            var clients = new List<INetChannel>();
-
-            ClientDistanceToList(source, VoiceRange, clients);
-
             message = SanitizeMessageCapital(source, message);
 
             var listeners = EntitySystem.Get<ListeningSystem>();
             listeners.PingListeners(source, message);
 
             message = FormattedMessage.EscapeText(message);
 
+            var clients = new List<ICommonSession>();
```

Comment:
```suggestion
            var sessions = new List<ICommonSession>();
```
Rename to sessions for consistency and readability.

Reply:
Resolved in commit #3269225